sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -171,7 +169,15 @@ internal class RadarLocationManager(
 
     internal fun handleBootCompleted() {
         logger.d("Handling boot completed")
+        wakeUp()
+    }
+
+    internal fun handleTaskRemoved() {
+        logger.d("Handling task removed")
+        wakeUp()
+    }
 
+    private fun wakeUp() {
         this.started = false
         RadarState.setStopped(context, false)
 
```

Comment:
do we want to add a log in the `addOnSuccessListener` to see what the last location requested was?

Reply:
I don't think that's necessary. The previous code logs the event that occurred to cause a call to `updateTracking`, and `updateTracking` logs the location.